### ISSUE TITLE
Better handling of MediaWiki File: links

### ISSUE THREAD

**Issue by ThomasWaldmann** [moinwiki/moin#144]
**[Original report](https://bitbucket.org/thomaswaldmann/moin-2.0/issue/144) by Shitiz Garg (Bitbucket: [Dragooon](https://bitbucket.org/Dragooon), GitHub: [Dragooon](https://github.com/Dragooon)).**

----------------------------------------

Currently MoinMoin just creates a <object> with xlink to the specified file, which is completely different from how MediaWiki handles it.

See http://www.mediawiki.org/wiki/Help:Images

Basically MediaWiki embeds an image with URL to the file, along with the ability to add caption, specify width/height, frame/border and a couple of other options.
